charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/values.yaml

```diff
@@ -54,3 +54,4 @@ configmapReload:
     tag: v0.3.0-rancher3
     pullPolicy: IfNotPresent
   resources: {}
+
```

Comment:
What about fluentd-tester ? See https://github.com/rancher/system-charts/pull/390/commits/4d4e3c2d419920f9559c1d1a90374e18835dd015#diff-7fbe58cc263dd407812a62df27b04a8a2b700eb11e30a24f8d6d1744c2e43dcfR5

Reply:
@nickgerace I assume we need to bump this so tester and real fluentD don't act differently, even though this specific bug probably won't show up there

Reply:
Ah tester would actually be necessary since it is specifically for fluentd Linux, if I recall correctly. @cbron

Reply:
Updated